poetry.lock

```diff
@@ -1175,6 +1189,24 @@ files = [
 Django = ">=2.2"
 redis = ">=3.0.0"
 
+[[package]]
```

Comment:
Strange? Cause `django-template-partials` is added in poetry.locks...
Can't explain right now